Resources/Prototypes/Maps/asterisk.yml

```
@@ -1,73 +1,84 @@
-#             Brigmedic: [ 1, 1 ]
-#             SecurityOfficer: [ 2, 3 ]
-#             SecurityCadet: [ 1, 3 ]
-#             Prisoner: [ 1, 2 ] # :^)
-#           #supply
-#             Quartermaster: [ 1, 1 ]
-#             MailCarrier: [ 1, 2 ]
-#             SalvageSpecialist: [ 2, 3 ]
-#             CargoTechnician: [ 2, 4 ]
-#           #civilian
-#             Passenger: [ -1, -1 ]
+ - type: gameMap
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)